src/components/dashboard/main-content/course-enrollments/course-cards/CompletedCourseCard.jsx

```diff
@@ -34,28 +35,37 @@ const CompletedCourseCard = (props) => {
     );
   };
 
+  // TODO: remove this function on/after December 15th, 2020 to only return
+  // the ``CertificateImg``, which has the new logo. until then, this function
+  // will show the image with the current logo and automatically switch to the
+  // image with the new edX logo on December 15th.
+  const getVerifiedCertificateImg = () => {
+    const now = new Date();
+    if (now < new Date('2020-12-15')) {
```

Comment:
this is a quick fix to get around that there's an image with the new edX logo that can't be shown until on/after December 15.